.cirrus.yml

```diff
@@ -367,3 +355,37 @@ task:
         - ./script/tool_runner.sh native-test --macos
       drive_script:
         - ./script/tool_runner.sh drive-examples --macos --exclude=script/configs/exclude_integration_macos.yaml
+
+# Intel macOS tasks.
+task:
+  << : *MACOS_INTEL_TEMPLATE
+  << : *FLUTTER_UPGRADE_TEMPLATE
+  matrix:
+    ### iOS+macOS tasks ***
+    # TODO(stuartmorgan): Move this to M1 once
+    # https://github.com/flutter/flutter/issues/103515 is resolved.
```

Comment:
The Flutter 3.0 arm64 version of macOS framework has been published, can we try on arm64 again?
https://github.com/CocoaPods/Specs/blob/master/Specs/4/2/c/FlutterMacOS/3.0.0/FlutterMacOS.podspec.json

Reply:
I just ran locally and still got `FlutterMacOS (2.10.2)`. Maybe it's not propagated yet?

Reply:
That's possible, sometimes it takes awhile. I'll check locally tomorrow.

Reply:
Turns out I had to `pod repo update`.